src/plone/rest/patches.zcml

```diff
@@ -13,4 +13,12 @@
     preserveOriginal="true"
     />
 
+  <monkey:patch
```

Comment:
@lukasgraf Plone 5.2 will run on WSGI. I have no idea how well this works with WSGI. Any idea @davisagli @jensens @pbauer?

Reply:
@tisto The patch isn't necessary any more for newer versions of `ZPublisher` anyway, because the status codes code [moved to the `zExceptions` module](https://github.com/zopefoundation/zExceptions/blame/master/src/zExceptions/__init__.py#L60) and already include the 308 that we need. The patching function is written in a way where it does nothing in this case.

However, I don't think the patch is even needed for 5.1 anymore - I could verify what exact Plone version is the highest that even needs the patch, and then only patch conditionally using a ZCML condition for the relevant (and confirmed working) Plone versions.